packages/protocol/contracts/common/Accounts.sol

```diff
@@ -180,6 +195,59 @@ contract Accounts is
     emit AccountMetadataURLSet(msg.sender, metadataURL);
   }
 
+  /**
+   * @notice Set the indexed signer for a specific role
+   * @param signer the address to set as default
+   * @param role the role to register a default signer for
+   */
+  function setDefaultSigner(address signer, string memory role) public {
+    require(
+      isGenericSigner(msg.sender, signer, role),
+      "Must finish authorization before setting default signer"
+    );
+
+    Account storage account = accounts[msg.sender];
+    if (keccak256(abi.encodePacked(role)) == keccak256(abi.encodePacked(VoteSigner))) {
```

Comment:
if we are going to be computing hashes of `role`, shouldn't we store them as `bytes32`?

Reply:
the rightside of these `==` comparisons  can be a `constant` either way

Reply:
Agree on storing as `bytes32` and offering versions of methods that pass the role as the hash already

Reply:
Makes sense